tests/test_phase_psf.py

```diff
@@ -403,21 +403,56 @@ def test_scale_unit():
 def test_stepk_maxk():
     """Test options to specify (or not) stepk and maxk.
     """
+    import time
     aper = galsim.Aperture(diam=1.0)
     rng = galsim.BaseDeviate(123456)
     # Test frozen AtmosphericScreen first
     atm = galsim.Atmosphere(screen_size=30.0, altitude=10.0, speed=0.1, alpha=1.0, rng=rng)
     psf = galsim.PhaseScreenPSF(atm, 500.0, aper=aper, scale_unit=galsim.arcsec)
-    stepk = psf.stepk
-    maxk = psf.maxk
+    t0 = time.time()
+    stepk1 = psf.stepk
+    maxk1 = psf.maxk
+    t1 = time.time()
+    print('stepk1 = ',stepk1)
+    print('maxk1 = ',maxk1)
+    print('t1 = ',t1-t0)
 
+    psf._prepareDraw()
+    stepk2 = psf.stepk
+    maxk2 = psf.maxk
+    t2 = time.time()
+    print('stepk2 = ',stepk2)
+    print('maxk2 = ',maxk2)
+    print('t2 = ',t2-t1)
+
+    # stepk is only accurate to about 25%.  maxk is quite a bit better.
```

Comment:
I'd guess that the stepk accuracy will get worse here if you increase the aperture diameter so that we're even further away from diffraction limited.